homeassistant/components/modbus.py

```diff
@@ -33,10 +34,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 SERVICE_WRITE_REGISTER = "write_register"
+SERVICE_WRITE_REGISTERS = "write_registers"
 
 ATTR_ADDRESS = "address"
 ATTR_UNIT = "unit"
 ATTR_VALUE = "value"
+ATTR_VALUES = "values"
```

Comment:
Actually I mean both. We already accept one or multiple entity_ids for `entity_id` attribute eg. I suggest remove `ATTR_VALUES` and only use `ATTR_VALUE`. Use one service that checks for a list passed to `ATTR_VALUE`. If it's a list use `HUB.write_registers`, if not use `HUB.write_register`.  I think you should also use YAML list syntax to pass multiple values as a list to `ATTR_VALUE`.